Tavisca.Bootcamp.LanguageBasics.Exercise1/Equation.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+namespace Tavisca.Bootcamp.LanguageBasics.Exercise1{
+class Equation{
+    public Equation(string operand1,string operand2,string operation,string result){
+        Operand1=operand1;
+        Operand2=operand2;
+        Operation=operation;
+        Result=result;
+    }        
+    public string Operand1 { get; private set; }
+    public string Operand2 { get; private set; }
+    public string Operation { get; private set; }
+    public string Result { get; private set; }
+    private bool IsValidValue(string actualValue,string expectedValue){
+        if(string.IsNullOrEmpty(actualValue)||string.IsNullOrEmpty(expectedValue))
+            return false;
+        if(expectedValue.Length==actualValue.Length)
+            return true;
+        return false;
+    }
+    private EquationParts GetEquationPartToBeCorrected(){
+        if(Operand1.Contains('?'))
+            return EquationParts.Operand1;
+        else if(Operand2.Contains('?'))
+            return EquationParts.Operand2;
+        else if(Result.Contains('?'))
+            return EquationParts.Result;
+        else
+            return EquationParts.Invalid;
+    }
+    private bool IsValidEquation(string operand1, string operand2, string result){
+        if(string.IsNullOrEmpty(operand1)||string.IsNullOrEmpty(operand2)||string.IsNullOrEmpty(result))
+            return false;
+        if(Convert.ToInt32(result)==Convert.ToInt32(operand1)*Convert.ToInt32(operand2))
+            return true;
+        return false;
+    }
+    private int MatchAndReturnFixedDigit(string actual, string expected){
+        var index = actual.IndexOf('?');
+        return int.Parse(expected[index].ToString());
+    }
+    public int CorrectEquationAndReturnMissingDigit(){
+        var needsCorrection=GetEquationPartToBeCorrected();
+        switch(needsCorrection){
+            case EquationParts.Operand1:
+                var ExpectedResult=Convert.ToInt32(Result)/Convert.ToInt32(Operand2);
```

Comment:
naming convention : local variable always starts with lower case , so it should be 'expectedResult'